mu-plugins/central-hub/tests/phpunit/unit/config-store/_theStore.php

```diff
@@ -107,4 +107,25 @@ public function test_should_return_stored_config_when_given_valid_store_key() {
 		// Clean up.
 		_the_store( __METHOD__, null, true );
 	}
+
+	/**
+	 *  Test _the_store() should overwrite a stored config using the same key.
+	 */
+	public function test_should_overwrite_a_stored_config_using_same_key() {
+		$config = [
+			'aaa' => 'bbb',
+			'ccc' => 'ddd'
+		];
+
+		$this->assertTrue( _the_store( 'foo', $config ) );
+
+		$new_config = [
+			'aaa' => 37,
+			'ccc' => 'Coding is fun!',
+			'eee' => 'WordPress rocks!',
+		];
+
+		$this->assertTrue( _the_store( 'foo', $new_config ) );
+	}
```

Comment:
@rgadon107 The test validates that the config was stored.  But it doesn’t validate the config was overwritten with the new config. You need to go one more step and get the config out of the store.  Make sure it actually overwrote it.

Reply:
@hellofromtonya 
>But it [the test] doesn’t validate the config was overwritten with the new config. 

You're right. When I went to bed last night, I knew I hadn't conclusively demonstrated that the config was overwritten. See next commit below (344d351), which addresses your comment.